app/_components/TestComponent/BadComponent.tsx

```
@@ -0,0 +1,60 @@
+import React, { useState, useEffect, useRef, Component } from "react";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)